workflows/prognostic_c48_run/tests/_regtest_outputs/test_regression.test_fv3run_checksum_restarts[nudging].out

```diff
@@ -1 +1 @@
-566da3dda9be0fd64285cec8f3b93bed
+62a2754f455f3bd8764fb74f78b4770e
```

Comment:
because the value of `gravity` used to compute net moistening has changed, the surface precipitation in a nudged or prognostic run changes, and I think this is why the checksum for the nudging restart file changes. I'm not sure why the restart checksum does not change for the sklearn test...

Reply:
It could be the case that the mock sklearn model only acts to moisten the atmosphere, in which case the surface precipitation change due to ML would be zero.

Reply:
Would be nice to move the constant change to another PR, so I can be certain that the refactors didn't break anything.

Reply:
Yeah, that's a good idea, I can isolate the gravity constant change to it's own PR.